charts/sourcegraph/values.yaml

```diff
@@ -615,6 +615,45 @@ minio:
   # -- PVC Storage Request for `minio` data volume
   storageSize: 100Gi
 
+otelCollector:
+  gateway:
+    # -- Name used by resources. Does not affect service names or PVCs.
+    name: "otel-collector"
+    image:
+      # -- Docker image tag for the `otel-collector` image
+      defaultTag: 3.42.2@sha256:xxxxxxx # TODO: there is no 3.42.2 image, so needs to be corrected with release of 3.43.0
```

Comment:
A 3.43.0 image has now been cut, so when the chart is bumped to that version this can be updated (as well as for the agent)